src/.editorconfig

```diff
@@ -337,3 +337,7 @@ dotnet_naming_style.begins_with_i.required_prefix = I
 dotnet_naming_style.begins_with_i.required_suffix =
 dotnet_naming_style.begins_with_i.word_separator =
 dotnet_naming_style.begins_with_i.capitalization = pascal_case
+
+# Justification: We expose CancellationToken on the public API to be consistent with V8 but can't pass it down
```

Comment:
@tmasternak @SzymonPobiega @timbussmann @danielmarbach you can't change the top level `.editorconfig`. RepoStandards will revert it - https://github.com/Particular/NServiceBus.TransactionalSession/pull/92